libs/auth/certs/dev/vx-admin-cert-authority-cert.pem

```diff
@@ -1,13 +1,13 @@
 -----BEGIN CERTIFICATE-----
```

Comment:
Just re-ran my handy `./scripts/generate-dev-keys-and-certs` script to update these! This will require anyone who has set up cards to re-run the `./scripts/configure-dev-java-card` script, but I think that's just me 😝